ui/components/multichain/pages/send/index.scss

```diff
@@ -0,0 +1,3 @@
+.multichain-send-page {
+  width: 408px;
+}
```

Comment:
Should we include the height css in this skeleton as well?

Reply:
Nope, the height will stretch to what's available, since height will vary based on contents.